Packs/SekoiaXDR/Integrations/SekoiaXDR/SekoiaXDR.py

```diff
@@ -610,7 +626,188 @@ def fetch_incidents(
+    )
+
+    demisto.debug(
+        f"Alert {alert_short_id} with status {alert_status} : last_update is {last_update} , alert_last_update is {alert_last_update}"  # noqa: E501
+    )
+
+    entries = []
+
+    # Add the events to the alert
+    if mirror_events and alert["status"]["name"] not in ["Closed", "Rejected"]:
+        earliest_time = alert["first_seen_at"]
+        lastest_time = "now"
+        term = f"alert_short_ids:{alert['short_id']}"
+        interval_in_seconds = INTERVAL_SECONDS_EVENTS
+        timeout_in_seconds = TIMEOUT_EVENTS
+
+        args = {
+            "earliest_time": earliest_time,
+            "lastest_time": lastest_time,
+            "query": term,
+            "interval_in_seconds": interval_in_seconds,
+            "timeout_in_seconds": timeout_in_seconds,
+        }
+        events = search_events_command(args=args, client=client)
+        alert["events"] = events.outputs  # pylint: disable=E1101
+
+    # Add the kill chain information to the alert
+    if mirror_kill_chain and alert["kill_chain_short_id"]:
+        try:
+            kill_chain = client.get_kill_chain(
+                kill_chain_uuid=alert["kill_chain_short_id"]
+            )
+            alert["kill_chain"] = kill_chain
+        except Exception as e:
+            # Handle the exception if there is any problem with the API call
+            demisto.debug(f"Error fetching kill_chain : {e}")
+
+    # This adds all the information from the XSOAR incident.
+    demisto.debug(
+        f"Alert {alert_short_id} with status {alert_status} have this info updated: {alert}"
+    )
+
+    investigation = demisto.investigation()
+    demisto.debug(f"The investigation information is {investigation}")
+
+    incident_id = investigation["id"]
+    incident_status = investigation["status"]
+
+    demisto.debug(
+        f"The XSOAR incident is {incident_id} with status {incident_status} is being mirrored with the alert {alert_short_id} that have the status {alert_status}."  # noqa: E501
+    )
+
+    # Close the XSOAR incident using mirroring
+    if (
+        (close_incident)
+        and (alert_status in ["Closed", "Rejected"])
+        and (investigation["status"] != 1)
+    ):
+        demisto.debug(
+            f"Alert {alert_short_id} with status {alert_status} was closed or rejected in Sekoia, closing incident {incident_id} in XSOAR"  # noqa: E501
+        )
+        entries = [
+            {
+                "Type": EntryType.NOTE,
+                "Contents": {
+                    "dbotIncidentClose": True,
+                    "closeReason": f"{alert_status} - Mirror",
+                    "closeNotes": close_note,
+                },
+                "ContentsFormat": EntryFormat.JSON,
+            }
+        ]
+
+    # Reopen the XSOAR incident using mirroring
+    if (
+        (reopen_incident)
+        and (alert_status not in ["Closed", "Rejected"])
+        and (investigation["status"] == 1)
```

Comment:
```suggestion
        and (investigation["status"] != 1)
```
When the status of the XSOAR incident is 1, it means that the incident is already Active and not Closed (the status will be 2 in this case), so if `investigation["status"] == 1` it means that the incident was already open, no need to reopen it again.